app/experimenter/nimbus-ui/src/index.test.tsx

```diff
@@ -16,7 +17,9 @@ describe("index.tsx", () => {
     require("./index");
     expect(ReactDOM.render).toHaveBeenCalledWith(
       <React.StrictMode>
-        <App />
+        <AppErrorBoundary>
```

Comment:
Bleh, I took `.toHaveBeenCalledWith` out of this test in [another PR](https://github.com/mozilla/experimenter/pull/3679/files#diff-824146ab1d2c016a050b808c1082ac96R21), seemed fragile to match _exactly_ what we're outputting here since the App has to be wrapped with Apollo Client - `MockedProvider`  which is the recommendation for mocking it doesn't work there and, passing in an identical Apollo Client to the one ran in the index in the test file still makes this test fail.

I'm still checking we're using React Strict mode in that PR and I think it's worth ensuring the index file renders the `AppErrorBoundary`. It looks like [Enzyme might have something](https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/contains.html) we can check the children with which if that works for React.Strict too would probably be better than what I'm doing in my PR or `.toHaveBeenCalledWith`, but we haven't needed Enzyme in `fxa-settings` or `fxa-payments`. 🤔

Reply:
Seems like maybe these should be separate tests? One for strict mode with the mocked render and another more like the old test? Switching to Enzyme might not be worth it, since I think we don't really want to mix react-testing-library and Enzyme

Reply:
Ah ok, I didn't realize using Enzyme meant "switching to" because of the different test environments.

I'm not sure we can do the "old test" with `.toHaveBeenCalledWith` and Apollo Client or at least, I spent a decent amount of time trying to get that to pass and it refused. I even copied and pasted the "expected" and "received" output in a diff checker to see what character was different and it came back exactly identical, yet still didn't pass. I could try again with fresh eyes🤷‍♀️